service/history/task/transfer_active_task_executor_test.go

```diff
@@ -893,8 +892,8 @@ func (s *transferActiveTaskExecutorSuite) expectTerminateRequest(childDomainName
 		) error {
 			s.Equal(childDomainID, request.DomainUUID)
 			s.Equal(childDomainName, request.TerminateRequest.Domain)
-			errors := []error{nil, &types.EntityNotExistsError{}}
-			return errors[rand.Intn(len(errors))]
+			errs := []error{nil, &types.EntityNotExistsError{}}
+			return errs[rand.Intn(len(errs))]
```

Comment:
I think this is a bad change - lets not do that.

Reply:
@jakobht This change only rename the local variable `errors` because it collides with the imported package `errors`.